src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -556,6 +556,122 @@ Here are the explanation for the example template above:
 * The `http-request auth realm kibana_users if !kibana_basic_auth` will trigger the HTTP basic auth prompt if the HTTP basic auth is not supplied with the request.
 * The example has list of users declared with insecure password. In the actual production scenario, consider using a `password` HAProxy directive which declares a secure password - refer to the [HAProxy](http://www.haproxy.org/) documentation for further details.
 
+### Securing the ConductR CLI with Basic Authentication
+
+HTTP Basic Authentication may be used to restrict access to a ConductR's [Control API](ControlAPI). The [ConductR CLI](CLI) uses the Control API to load bundles and retrieve cluster information, thus applying Basic Authentication against this service restricts who is allowed to load, start and stop bundles in the ConductR cluster. This provides another method of securing the cluster beyond security group and firewall restrictions. This is particularly useful when a fully restricted [bastion host](https://en.wikipedia.org/wiki/Bastion_host) is not viable, or to further restrict access to particular clusters via the CLI. For example, while it may be acceptable to allow everyone on a team to deploy bundles to a development or staging cluster, a production cluster may require more restrictive access.
+
+As a prerequisite, the ConductR Cores should be configured to bind to a private IP address and an available port. By default, the ConductR Cores will bind to the configured `CONDUCTR_IP` and port `9005`. Binding to a different IP and port is achieved by adding the following lines to `/usr/share/conductr/conf/conductr.ini` and restarting the ConductR Core Service. For this case, we will bind to localhost and use port 9055 internally, such that the ConductR Control API will only be available locally on this host.
+
+```
+-Dconductr.control-server.ip = 127.0.0.1
+-Dconductr.control-server.port = 9055
+```
+
+Restart the ConductR Core Service with the following command:
+
+```
+> sudo service conductr restart
+```
+
+HAProxy must then be configured to provide Basic Authentication and SSL termination for the Control Protocol, such that it is still accessed via the default port `9005` externally. As we bound the Control Protocol to localhost above, we are assuming HAProxy is running on the same host(s) as the ConductR Core(s).
+
```

Comment:
This approach won't work. You cannot assume that HAProxy and core are in the same node. Drop the 127.0.0.1 idea and configure HAProxy with the static location of, say, 3 cores. They won't move much.